portfolio/src/main/webapp/script.js

```diff
@@ -27,6 +27,33 @@ function getInfo(){
     })
 }
 
+function displayComment(){
+    console.log("function is running");
```

Comment:
Please either remove this log line or say something more specific like "Displaying comments"